app/lib/database.py

```diff
@@ -49,99 +48,131 @@ def __init__(self, cursor):
         assert(cursor is not None)
         self._cursor = cursor
 
-    def create_json_table(self, table_name: str):
+    def create_metadata_file_view(self, table_name: str, file_type: str):
```

Comment:
What do you think about pulling the functions for metadata_files and bundles into their own separate classes?